.github/dependabot.yml

```diff
@@ -26,15 +26,11 @@ updates:
       # https://github.com/FlowCrypt/flowcrypt-browser/pull/3961#issuecomment-921335748
       - dependency-name: "iso-8859-2"
         versions: [ ">=1.0.1" ]
-      - dependency-name: "@openpgp/web-stream-tools"
-        versions: [">= 0.0.14"]
       - dependency-name: "ava"
         update-types: ["version-update:semver-major"]
       - dependency-name: "chai"
         update-types: ["version-update:semver-major"]
       - dependency-name: "chai-as-promised"
         update-types: ["version-update:semver-major"]
-      - dependency-name: "openpgp"
```

Comment:
Could you tell me the reason to remove openpgp from dependabot?

Reply:
I removed it from `ignore-updates` list, as previously we disabled it's upgrade to v6 to fix compatibility issues.
But as currently we use the latest `openpgpjs` version, there is no reason to have it in ignore list.

Reply:
Ohhh, yep, right.
I missed ignore part.
Thank you for notice